models/users.js

```diff
@@ -933,7 +933,7 @@ if (Meteor.isServer) {
       user.authenticationMethod = 'oauth2';
 
       // see if any existing user has this email address or username, otherwise create new
-      const existingUser = Meteor.users.findOne({
+      const existingUser = Users.findOne({
```

Comment:
@salleman33 Can you please explain what's the benefit of this change? If I understand the code correctly, `Users` is simply a reference to `Meteor.users` (see https://github.com/wekan/wekan/blob/master/models/users.js#L7).

Do you prefer `Users` since the string length is shorter?

Reply:
@marc1006 Currently I'm in progress of reverting these changes.

Reply:
@marc1006 Because comments are not visible anymore in mobile view #3182

Reply:
Yes, this commit is useless. I did'nt know that Users equals Meteor.users. I thought it was different.
You can revert it .